### PR TITLE
use testFile in some tests

### PR DESCRIPTION
**Pull request by ryan-williams**

### CODE REVIEW

File: adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala [hunk 1]
```diff
@@ -17,18 +17,16 @@
  */
 package org.bdgenomics.adam.cli
 
-import com.google.common.io.Files
 import java.io.File
-import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.rdd.ADAMContext._
+
```
Comment (fnothaft) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: Can you clean up the newline here?

File: adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala [hunk 2]
```diff
@@ -17,18 +17,16 @@
...
+import com.google.common.io.Files
```
Comment (fnothaft) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: Are we still importing `Files` here?

Comment (ryan-williams) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: not sure I understand the q; `Files` is still used in this file.

File: adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala (hunk 1 above)
Comment (ryan-williams) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: mm my IntelliJ's ⌘⇧O defaults to a blank line between `java` imports, others, and `scala` imports:



I feel like that's a style I've seen around out in the world. I guess ADAM is not set up that way, but it also seems like people are not using IntelliJ's (or anything else's) import-optimization helpers in the codebase, as I've seen multiple unused imports laying around.

Also I'm not sure if I can change that IntelliJ setting per-project :( but lmk if given all this you still want to just have "one import blob" be the rule.

Comment (heuermh) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: I admit to opening up IntelliJ only rarely, and am likely the culprit for some of those unused imports.  If you have settings that work well with our style, maybe we could add them to the contributing doc.

File: adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala (hunk 2 above)
Comment (fnothaft) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: Yeah, I just wanted to confirm that.

File: adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala (hunk 1 above)
Comment (fnothaft) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: > Also I'm not sure if I can change that IntelliJ setting per-project :( but lmk if given all this you still want to just have "one import blob" be the rule.

That would be my preference for consistency with the remainder of the codebase. If it's a big hassle from your side, I can fix on merge.

Comment (ryan-williams) [inline adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala]: @fnothaft I changed it

Intersting @heuermh, what do you edit in?

**Comment by fnothaft** [comment]
I can't speak for @heuermh but I never use Intellij either and do all my editing in emacs. I tried switching to Intellij several years back but it's emacs compat mode is pretty rough. I code in a variety of other languages, so cutting over and switching to Intellij doesn't make sense as an investment for me.

**Comment by fnothaft** [comment]
Will merge on tests rerunning and passing.

**Comment by heuermh** [comment]
Emacs and whatever scala mode comes in via melpa for me.  I try to look at all my prs in IntelliJ for the static code checking at least once, but the way it handles git makes me nervous, so I don't use it full time.  And I prefer running in terminals.  :)

**Comment by fnothaft** [comment]
Jenkins, retest this please.

**Comment by ryan-williams** [comment]
why retest OOC?

**Comment by fnothaft** [comment]
@ryan-williams I just merged #1135 which was +~2500 -~600 so I am somewhat distrustful of old Jenkins results is all.

**Comment by fnothaft** [comment]
Merged! Thanks @ryan-williams!

**Pull request merged**